Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -164,6 +166,28 @@ namespace AzToolsFramework::ViewportUi::Internal
         }
     }
 
+    const bool ViewportUiDisplay::HideSwitcher(ViewportUiElementId switcherId)
+    {
+        if (auto switcher = qobject_cast<ViewportUiSwitcher*>(GetViewportUiElement(switcherId).get()))
```

Comment:
nit: this (and below) could be `const` (also be good to add `auto*` for clarity)